platform_versioned_docs/version-24.1/secrets/overview.mdx

```diff
@@ -75,26 +75,26 @@ The ECS Agent uses the [Batch Execution role](https://docs.aws.amazon.com/batch/
 **IAM permissions**
 
 1. Add the [`AmazonECSTaskExecutionRolePolicy` managed policy](https://docs.aws.amazon.com/aws-managed-policy/latest/reference/AmazonECSTaskExecutionRolePolicy.html).
-2. Add this inline policy:
+1. Add [this inline policy](https://github.com/seqeralabs/nf-tower-aws/blob/master/launch/secrets-policy-execution-role.json):
 
 ```json
     {
         "Version": "2012-10-17",
         "Statement": [
             {
                 "Sid": "AllowECSAgentToRetrieveSecrets",
-                "Action": [
-                    "secretsmanager:GetSecretValue"
-                ],
-                "Resource": [
-                    "arn:aws:secretsmanager:<YOUR_COMPUTE_REGION>:*:secret:*"
-                ],
-                "Effect": "Allow"
-            }
-        ]
+                "Effect": "Allow",
+                "Action": "secretsmanager:GetSecretValue",
+                "Resource": "arn:aws:secretsmanager:*:*:secret:tower-*"
+        }
+      ]
     }
 ```
 
+:::note
+Including `tower-*` in the Resource ARN above limits access to Platform secrets only (as opposed to all secrets in the given region).
```

Comment:
Remove `in the given region` because your no longer asking customers to limit to a specific region.

```suggestion
Including `tower-*` in the Resource ARN above limits access to Platform secrets only (as opposed to all secrets across all regions).
```